processing/src/main/java/io/druid/query/Queries.java

```diff
@@ -64,6 +68,40 @@ public String apply(AggregatorFactory input)
         );
         combinedAggNames.add(postAgg.getName());
```

Comment:
Modified it a bit. I can actually do the check as a feature of `Set.add` (see code)